recipes/dm-tree/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.1.5" %}
+
+package:
+  name: dm-tree
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/70/0a/bc3e9865603332c525fc218aceb023762aeffc2a86ff99b347b67ee3f2a8/dm-tree-{{ version }}.tar.gz
```

Comment:
Can you use the PyPI URL without the hash in the URL? This would be better for getting automated  version updates by the bot. Probably the most convenient approach would be to regenerate this recipe using https://github.com/conda-incubator/grayskull

Reply:
I've made a new `meta.yaml` and then manually merged it with current one, hope it's better now.